datafusion/core/tests/parquet/filter_pushdown.rs

```diff
@@ -240,100 +241,151 @@ async fn single_file_small_data_pages() {
     // docker run -v /tmp:/tmp nathanhowell/parquet-tools dump -d -c pod -n /tmp/.tmppkTohR/data.parquet
     //
     // ```
-    // row group 0
-    //     --------------------------------------------------------------------------------
-    //     pod:  BINARY UNCOMPRESSED DO:782 FPO:1215 SZ:744/744/1.00 VC:2048 ENC:RLE,RLE_DICTIONARY,PLAIN ST:[min: azvagebjesrqboyqxmgaskvpwddebuptqyy, max: zamirxzhihhfqdvhuxeziuukkqyutmczbhfgx, num_nulls not defined]
+    //  row group 0
```

Comment:
Thank you for updating these values. I think it would make the PR easier to review if we can avoid changing the existing tests if possible (I left a suggestion on the data generator about a potential way to do this)